tests/integration/test_api/test_config/test_max_upload_size/test_max_upload_size.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+from os.path import join, dirname, realpath
+from uuid import uuid4
+
+import pytest
+import requests
+
+from wazuh_testing.tools.configuration import check_apply_test, get_api_conf
+
+# Marks
+
+pytestmark = pytest.mark.server
```

Comment:
It would be useful to add the tier (0, 1, or 2) of the test. For example:

```suggestion
pytestmark = [pytest.mark.server, pytest.mark.tier(level=0)]
```